File: 2988/2988.py
class Solution:
    def manager_of_the_largest_department(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        return 1 + max(self.manager_of_the_largest_department(root.left), self.manager_of_the_largest_department(root.right))

File: 2988/2988.go
func managerOfTheLargestDepartment(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := managerOfTheLargestDepartment(root.Left)
	r := managerOfTheLargestDepartment(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}